presto-main/src/main/java/com/facebook/presto/type/DoubleOperators.java

```diff
@@ -175,7 +175,7 @@ public static long castToInteger(@SqlType(StandardTypes.DOUBLE) double value)
             return Ints.checkedCast((long) MathFunctions.round(value));
         }
         catch (IllegalArgumentException e) {
-            throw new PrestoException(NUMERIC_VALUE_OUT_OF_RANGE, e);
+            throw new PrestoException(NUMERIC_VALUE_OUT_OF_RANGE, "Out of range for integer: " + value, e);
```

Comment:
Not necessarily because since we have FLOAT which is displayed the same way as DOUBLE "from double" gives some information. But since we'll be working on tracking errors back to the original query position, it's not very important now.